### PR TITLE
Add PShow instances to print PValue and PData values

### PR DESCRIPTION
**Pull request by facundominguez**

### CODE REVIEW

**Comment by facundominguez** [comment]
Hello! This PShow instances are being useful to debug plutarch programs, I thought of sharing it here in case you want it in.

I'm testing it on a fork of plutarch, so minor adaptations might be necessary to make it work here.

**Comment by L-as** [comment]
Thanks!

File: Plutarch/Api/V1/Value.hs [hunk 1]
```diff
@@ -123,6 +124,11 @@ newtype PValue (keys :: KeyGuarantees) (amounts :: AmountGuarantees) (s :: S)
   deriving anyclass (PlutusType, PIsData, PShow)
 instance DerivePlutusType (PValue keys amounts) where type DPTStrat _ = PlutusTypeNewtype
 
+instance PShow (PValue keys amounts) where
```
Comment (L-as) [inline Plutarch/Api/V1/Value.hs:127]: Why not just derive this?

Comment (L-as) [reply Plutarch/Api/V1/Value.hs:127]: Maybe you're unaware of #537 ?

Comment (facundominguez) [reply Plutarch/Api/V1/Value.hs]: > Maybe you're unaware of #537 ?

I was unaware indeed. I removed the related commits from this PR.

**Comment by L-as** [comment]
Thanks!

**Pull request merged**